modules/nw-rfhe-creating_bmc_event_sub.adoc

```diff
@@ -0,0 +1,164 @@
++
+[NOTE]
+====
+Enabling Redfish events on specific hardware is outside the scope of this information. For more information about enabling Redfish events for your specific hardware, consult the BMC manufacturer documentation.
+====
+
+.Procedure
+. Confirm that the node hardware has the Redfish `EventService` enabled by running the following `curl` command:
++
+[source,terminal]
+----
+curl https://<bmc_ip_address>/redfish/v1/EventService --insecure -H 'Content-Type: application/json' -u "<bmc_username>:<password>"
+----
++
+where:
++
+--
+bmc_ip_address:: is the IP address of the BMC where the Redfish events are generated.
+--
++
+.Example output
+[source,terminal]
+----
+{
+   "@odata.context": "/redfish/v1/$metadata#EventService.EventService",
+   "@odata.id": "/redfish/v1/EventService",
+   "@odata.type": "#EventService.v1_0_2.EventService",
+   "Actions": {
+      "#EventService.SubmitTestEvent": {
+         "EventType@Redfish.AllowableValues": ["StatusChange", "ResourceUpdated", "ResourceAdded", "ResourceRemoved", "Alert"],
+         "target": "/redfish/v1/EventService/Actions/EventService.SubmitTestEvent"
+      }
+   },
+   "DeliveryRetryAttempts": 3,
+   "DeliveryRetryIntervalSeconds": 30,
+   "Description": "Event Service represents the properties for the service",
+   "EventTypesForSubscription": ["StatusChange", "ResourceUpdated", "ResourceAdded", "ResourceRemoved", "Alert"],
+   "EventTypesForSubscription@odata.count": 5,
+   "Id": "EventService",
+   "Name": "Event Service",
+   "ServiceEnabled": true,
+   "Status": {
+      "Health": "OK",
+      "HealthRollup": "OK",
+      "State": "Enabled"
+   },
+   "Subscriptions": {
+      "@odata.id": "/redfish/v1/EventService/Subscriptions"
+   }
+}
+----
+
+. Get the {redfish-operator} service route for the cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get route -n openshift-bare-metal-events
+----
++
+.Example output
+[source,terminal]
+----
+NAME             HOST/PORT                                                       PATH   SERVICES                 PORT   TERMINATION   WILDCARD
+hw-event-proxy   hw-event-proxy-cloud-native-events.apps.compute-1.example.com          hw-event-proxy-service   9087   edge          None
+----
+
+. Create a `BMCEventSubscription` resource to subscribe to the Redfish events:
+
+.. Save the following YAML in the `bmc_sub.yaml` file:
++
+[source,yaml]
+----
+apiVersion: metal3.io/v1alpha1
+kind: BMCEventSubscription
+metadata:
+  name: sub-01
+  namespace: openshift-machine-api
+spec:
+   hostName: <hostname> <1>
+   destination: <proxy_service_url> <2>
+   context: ''
+----
+<1> Specifies the name or UUID of the worker node where the Redfish events are generated.
+<2> Specifies the bare-metal event proxy service, for example, `hw-event-proxy-cloud-native-events.apps.compute-1.example.com/webhook`.
```

Comment:
The url should include https://
Example: `https://hw-event-proxy-openshift-bare-metal-events.apps.compute-1.example.com/webhook`